src/Nri/Ui/MediaQuery/V1.elm

```diff
@@ -1,52 +1,75 @@
 module Nri.Ui.MediaQuery.V1 exposing
     ( mobile, notMobile
+    , mobileBreakpoint
```

Comment:
No big deal but I am curious about the need to expose the breakpoints. Do we expect other styling to be dependent upon the breakpoint pixel sizes?

Reply:
I expect other styling to be dependent on the breakpoint, yes!

Specifically, I think we often have layouts like this:

![image](https://user-images.githubusercontent.com/8811312/148849315-94638c96-9b85-46d7-b9f7-9b2c3f7c8bde.png)

Where there's an element that has a max-width or width equal to the page breakpoint. I think having the breakpoint and the max width be identical makes the resizing experience nicer. I could be wrong -- I just like the idea of the margin-auto decreasing until it can't decrease further, and then immediately transitioning into the mobile view

Reply:
I agree that is a usability win! I don't think there is anything else we should be doing different here. 

I was probing to see if we could figure out a way to encapsulate the pixel size as an implementation detail; exposing more narrowly typed and specifically named abstractions instead. I agree it is too early for that. Attempts at abstraction before we understand the pattern lead to over-engineering and all. I just thought I'd ask (for grins). 

The idea was that if we wanted to make an element max-width equal to a mobile breakpoint then we might offer 
```elm
maximizedMobileContainer : List (Attribute msg) -> List (Html msg) -> Html msg
```
where the function includes the attributes necessary (setting the max width). However, that looks clumsy to me. If you see a bunch of repeated patterns in how people consume the breakpoints then maybe there will come a time for a new abstraction. Or maybe not. 😄